gapis/capture/decoder.go

```diff
@@ -72,6 +72,7 @@ func (d *decoder) EndGroup(ctx context.Context, id uint64) error {
 
 	switch obj := obj.(type) {
 	case *cmdGroup:
+		obj.invoked = true
```

Comment:
This line is not actually needed, as the object falls out of scope in this function anyway. There's no harm in it though.